CMakeLists.txt

```diff
@@ -297,10 +299,20 @@ if (WITH_PYTHON)
     add_subdirectory(symengine/python)
 endif ()
 
+if (WITH_PYTHON)
+    message("Copying source of python wrappers into: ${CMAKE_CURRENT_BINARY_DIR}/symengine/python/symengine")
+    file(COPY symengine/python/symengine/ DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/symengine/python/symengine)
+endif()
```

Comment:
I just moved these lines up. When you build out of source, you need to copy the files, so that you can run the tests in the build directory directly.